### ISSUE TITLE
Priest spell Empowered Renew

### ISSUE THREAD

**Issue by Byssarod** [dalaranwow/dalaran-wow#5184]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Spell Empowered Renew in Holy talent tree is not healing for correct amount.
**Expected behaviour**: Tell us what should happen instead
Empowered Renew should heal  .5* (Rank 1)/ .10* (Rank 2) /.15 (Rank 3)* total periodic amount healed.
**Steps to reproduce the problem**:

1. Cast Renew with Empowered Renew talent
2. Calculate total periodic healing of Renew
3. Multiply total healing of Renew by .5/.10/.15 (Depend on rank), and see its actually less than it should be

**Include proofs for this behaviour**
http://wowwiki.wikia.com/wiki/Empowered_Renew
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
https://wotlk.evowow.com/?spell=63534 ; https://wowgaming.altervista.org/aowow/?spell=63534
https://wotlk.evowow.com/?spell=63542 ; https://wowgaming.altervista.org/aowow/?spell=63542
https://wotlk.evowow.com/?spell=63543 ; https://wowgaming.altervista.org/aowow/?spell=63543
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
https://imgur.com/a/D4hVp

**Comment by AttendantTock** [comment]
Hello @Byssarod and thanks for the report,

I do confirm this issue on the PTR, where the heal for rank 3 is slightly above 11%.
This stands for any combination of Improved Renew, Empowered Renew, Glyph of Renew, high or low +spell power.

<img width="484" alt="capture d ecran 2017-12-16 a 18 01 30" src="https://user-images.githubusercontent.com/29557022/34073153-362f0228-e28b-11e7-9a4f-3822c64979aa.png">

**Comment by robinsch** [comment]
Nope. Your Renew spell gains an additional $s1% of your bonus healing effects.

It just scales with your bonus healing. So it's defaultCoefficient * 1.15f

**Comment by Byssarod** [comment]
What? The report is about 2. part of that talent. The instant heal from renew, not the improve power of renew.

**Comment by Byssarod** [comment]
Im just curious, if someone is working on this fix since robinsch little misunderstand issue.
